include/openssl/des.h

```diff
@@ -148,8 +145,7 @@ int DES_check_key_parity(const_DES_cblock *key);
 int DES_is_weak_key(const_DES_cblock *key);
 /*
  * DES_set_key (= set_key = DES_key_sched = key_sched) calls
- * DES_set_key_checked if global variable DES_check_key is set,
- * DES_set_key_unchecked otherwise.
+ * DES_set_key_unchecked
```

Comment:
This comment is wrong. It now calls DES_set_key_checked.

Reply:
oops.  fix in #9292